.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# defaults
+[*]
+insert_final_newline = true     
+
+# 4 space indentation
+[*.{ini,py,py.tpl,rst}]
+indent_style = space
+indent_size = 4
+
+# 4-width tabbed indentation
+[*.{sh,bat.tpl,Makefile.tpl}]
```

Comment:
Again, there are no such files present. We can add such entries later once we have these files in the codebase. I hope you get the idea :)